src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.cs

```diff
@@ -775,5 +775,31 @@ protected internal override Delegate GetDataViewGetter(DataViewRowCursor cursor)
 
         private static ValueGetter<double> CreateDecimalValueGetterDelegate(DataViewRowCursor cursor, PrimitiveDataFrameColumn<decimal> column) =>
             (ref double value) => value = (double?)column[cursor.Position] ?? double.NaN;
+
+        protected internal override void AddValueUsingCursor(DataViewRowCursor cursor, DataViewSchema.Column column, Delegate getter)
```

Comment:
Is `column` going to be used? I don't see it used in either override.

Reply:
Nope. Removed it now. I missed it as I was refactoring (sigh).